YouTubeCrawler/src/main/scala/com/sofiia/production/akka/Workers.scala

```diff
@@ -0,0 +1,12 @@
+package com.sofiia.production.akka
+
+import akka.actor.{Actor, ActorRef}
+import com.sofiia.production.functions.YouTubeVideoCrawler
+
+class Workers(APIKey: String,  actorRef: ActorRef) extends Actor{
+  override def receive: Receive = {
+    case id: String =>
```

Comment:
and what if it will be not string or improper string? There are a lot of not implemented cases  here.